### ISSUE TITLE
Improve accessibility of badges

### ISSUE THREAD

**Issue by mairatma** [metal/metal-autocomplete-badges#2]
We need to improve the accessibility for this component, both by adding keyboard interactions and aria attributes.

Note that this issue should not try to improve accessibility of the autocomplete part, since that's handled by [metal-autocomplete](https://github.com/metal/metal-autocomplete), so it should be done there instead.

**Comment by mairatma** [comment]
Just to document the requirements of the expected keyboard interaction here.

Always when there are badges but there's nothing typed in the input at the moment:
- If no badge is focused:
  - Backspace or left arrow: focuses the previous badge, if any
  - Right arrow: focuses the next badge, if any
- If a badge is focused:
  - Backspace or enter: removes the focused badge

An example that works like this: http://sliptree.github.io/bootstrap-tokenfield/

**Comment by mairatma** [comment]
Just FYI, a pull request has been sent for this here: https://github.com/metal/metal-autocomplete-badges/pull/3.
